prime-router/src/main/kotlin/FakeReport.kt

```diff
@@ -111,7 +110,7 @@ class FakeReport(val metadata: Metadata) {
                     else -> TODO()
                 }
             }
-            Element.Type.TELEPHONE -> faker.numerify("##########:1:") // faker.phoneNumber().cellPhone()
+            Element.Type.TELEPHONE -> faker.numerify("2#########:1:") // faker.phoneNumber().cellPhone()
             Element.Type.EMAIL -> "${context.patientName.username()}@email.com"
```

Comment:
The bug here is that occasional leading zeroes break our phone number parser, when fake data is read back into the hub.    Since such numbers don't occur in real life, I just forced the first digit to be non-zero.